nix/overlays/checked-shell-script/checked-shell-script.nix

```diff
@@ -50,6 +56,10 @@ let
           trap 'rm -rf "$tmpdir"' SIGINT SIGTERM
         ''
 
+        + lib.optionalString (useEnv != false) ''
+          export PATH="${useEnv.env}/bin:$PATH"
+        ''
```

Comment:
This should be

```
env="$(cat ${useEnv.env})"
export PATH="$env/bin:$PATH"
```

The `$(cat ...)` part is significant

Reply:
I just realized that, too. :D

(latest force-push changed just that)

Reply:
The name `useEnv` seems a bit off... maybe `withEnv` or `withEnvOf` with a default of `null`?

Reply:
Cleanest option probably is `withEnv` with a default of `null` and pass in `postgrest.env` on the call sites

Reply:
@wolfgangwalther unresolving to make sure you saw the naming :)

Reply:
I went with that, much better. Will wait to push until the current CI run finishes, to hopefully get the first two CI runs in already.